src/styles/class-overrides.scss

```diff
@@ -97,6 +97,12 @@
   }
 }
 
+.usa-footer__return-to-top {
+  color: color("base-darker");
+  line-height: $mocto-box-height;
```

Comment:
I'm wondering why you set line-height here? Also in my most recent PR I got rid of the following in the mocto-theme file: 
```
$mocto-button-border-width: 2px;
$mocto-link-border-width: 4px;
$mocto-link-padding: 2px;
```
instead I use spacing tokens in custom overrides, like this: `padding-left: units("105");` I think it makes sense to use a spacing token here

Reply:
Got rid of `line-height`. I used spacing tokens in custom overrides otherwise.